package.json

```diff
@@ -58,5 +58,10 @@
   "engines": {
     "node": "^18.20.1",
     "pnpm": "^9.6.0"
+  },
+  "pnpm": {
+    "patchedDependencies": {
+      "minimist@1.2.8": "patches/minimist@1.2.8.patch"
```

Comment:
might be nice to name this something like `patches/minimist@1.2.8-allow-hex.patch`

Reply:
updated in https://github.com/latticexyz/mud/pull/3105